scripts/union-compare-two-tables.sql

```diff
@@ -0,0 +1,74 @@
+SELECT
```

Comment:
IIUC, when comparing, we should only compare the columns that existed previous to the migration. If so, can you document that?

Reply:
I added some comments in a subsequent commit that I'll squash later.